site/api/prisma/schema.prisma

```diff
@@ -162,23 +196,23 @@ model Appearance {
 }
 
 model Face {
-  id           Int                 @id @default(autoincrement())
+  id           Int         @id @default(autoincrement())
   x            Float
   y            Float
   height       Float
   width        Float
-  appearanceId Int?                @map("appearance_id")
-  addedById    Int?                @map("added_by_id")
-  createdAt    DateTime            @default(now()) @map("created_at")
-  updatedAt    DateTime            @default(now()) @updatedAt @map("updated_at")
-  score        Float               @default(0)
-  imageId      Int                 @map("image_id")
-  source       FaceSource          @default(Manual)
-  personId     Int?                @map("person_id")
-  addedBy      User?               @relation(fields: [addedById], references: [id])
-  appearance   Appearance?         @relation("apperance_face", fields: [appearanceId], references: [id])
-  image        Image               @relation(fields: [imageId], references: [id], onDelete: Cascade)
-  person       Person?             @relation("face_person", fields: [personId], references: [id])
+  appearanceId Int?        @map("appearance_id")
+  addedById    Int?        @map("added_by_id")
+  createdAt    DateTime    @default(now()) @map("created_at")
+  updatedAt    DateTime    @default(now()) @updatedAt @map("updated_at")
+  score        Float       @default(0)
+  imageId      Int         @map("image_id")
+  source       FaceSource  @default(Manual)
+  personId     Int?        @map("person_id")
```

Comment:
Generally, can we not call this person.. but Idol? Or Artist? Person soudns so iffy, and i think this is for kpop only, so we should stay with the general terms

Reply:
we use person everywhere. I want to avoid using kpop-specific terms in the backend and keep it kpop in the frontend so we can pivot more easily

Reply:
So you want all fields and names to be more agnostic? So we can easily expand into diff things like jpop, korean actors etc?

Reply:
yep, that's the idea